contracts/DCAStrategies/DCAStrategies/DCAStrategiesPermissionsHandler.sol

```diff
@@ -12,7 +13,11 @@ abstract contract DCAStrategiesPermissionsHandler is IDCAStrategiesPermissionsHa
     uint248 lastUpdated;
   }
 
-  // mapping(bytes32 => TokenPermission) public tokenPermissions; // key(id, operator) => TokenPermission
+  using PermissionMath for IDCAStrategies.Permission[];
+  using PermissionMath for uint8;
+
+  mapping(uint256 => uint256) public lastOwnershipChange;
+  mapping(bytes32 => TokenPermission) public tokenPermissions; // key(id, operator) => TokenPermission
```

Comment:
Could we make this internal and add a public function that takes the id + account and returns the `TokenPermission`? So it's easier to use from outside